runtime/lua/vim/lsp/util.lua

```diff
@@ -57,7 +57,12 @@ local function get_border_size(opts)
     end
     height, width = unpack(border_size[border])
   else
+    if 8 % #border ~= 0 then
+      error("floating preview border is not correct. Please refer to the docs |vim.api.nvim_open_win()|"
```

Comment:
```suggestion
      error("floating preview border is not correct. Refer to the docs |vim.api.nvim_open_win()|"
```

please is never needed in messages, it's noise

Reply:
But a "see" in front of the ":help" would read better.

Reply:
not really, that is not the convention in messages. The messages are seen many times and don't need to explain the same thing over and over. The convention is learned only once.